docs/deployment/cli.md

```diff
@@ -23,664 +23,254 @@ specific language governing permissions and limitations
+./bin/flink cancel $JOB_ID
 {% endhighlight %}
+```
+Cancelling job cca7bc1061d61cf15238e92312c2fc20.
+Cancelled job cca7bc1061d61cf15238e92312c2fc20.
+```
+The corresponding job's state will be transitioned from `Running` to `Cancelled`. Any computations 
+will be stopped.
 
-The run command has a savepoint flag to submit a job, which restores its state from a savepoint. The savepoint path is returned by the savepoint trigger command.
-
-By default, we try to match all savepoint state to the job being submitted. If you want to allow to skip savepoint state that cannot be restored with the new job you can set the `allowNonRestoredState` flag. You need to allow this if you removed an operator from your program that was part of the program when the savepoint was triggered and you still want to use the savepoint.
+#### Stop: Cancelling a Job but creating a final Savepoint
 
+Another action for stopping a job is `stop`. It function in the same way as `cancel` but creates a 
+savepoint as part of the cancellation process:
 {% highlight bash %}
-./bin/flink run -s <savepointPath> -n ...
+./bin/flink stop --savepointPath /tmp-flink-savepoints $JOB_ID
 {% endhighlight %}
+```
+Suspending job "cca7bc1061d61cf15238e92312c2fc20" with a savepoint.
+Savepoint completed. Path: file:/tmp/flink-savepoints/savepoint-cca7bc-bb1e257f0dab
+```
+We have to use `--savepointPath` to specify the savepoint folder if 
+[state.savepoints.dir]({% link deployment/config.md %}#state-savepoints-dir) isn't set.
 
-This is useful if your program dropped an operator that was part of the savepoint.
-
-#### Dispose a savepoint
+### Starting a Job from a Savepoint
 
+Starting a job from a savepoint can be achieved using the `run` (and `run-application`) action.
 {% highlight bash %}
-./bin/flink savepoint -d <savepointPath>
+./bin/flink run --detached --fromSavepoint /tmp/flink-savepoints/savepoint-cca7bc-bb1e257f0dab ./examples/streaming/StateMachineExample.jar
 {% endhighlight %}
+```
+Usage with built-in data generator: StateMachineExample [--error-rate <probability-of-invalid-transition>] [--sleep <sleep-per-record-in-ms>]
+Usage with Kafka: StateMachineExample --kafka-topic <topic> [--brokers <brokers>]
+Options for both the above setups:
+        [--backend <file|rocks>]
+        [--checkpoint-dir <filepath>]
+        [--async-checkpoints <true|false>]
+        [--incremental-checkpoints <true|false>]
+        [--output <filepath> OR null for stdout]
+
+Using standalone source with error rate 0.000000 and sleep delay 1 millis
+
+Job has been submitted with JobID 97b20a0a8ffd5c1d656328b0cd6436a6
+```
 
-Disposes the savepoint at the given path. The savepoint path is returned by the savepoint trigger command.
-
-If you use custom state instances (for example custom reducing state or RocksDB state), you have to specify the path to the program JAR with which the savepoint was triggered in order to dispose the savepoint with the user code class loader:
```

Comment:
Why did we drop this information? Is this no longer valid or too expert usage?

Reply:
Good catch 👍